app/hotels/src/map/allHotels/HotelSwipeItem.js

```diff
@@ -32,11 +32,21 @@ export class HotelSwipeItem extends React.Component<Props> {
 
   render = () => {
     const { width, data } = this.props;
-
+    const name = idx(data, _ => _.name);
+    const price = idx(data, _ => _.price);
+    const thumbnailUrl = idx(data, _ => _.mainPhoto.thumbnailUrl);
+    const stars = idx(data, _ => _.rating.stars);
+    const score = idx(data, _ => _.review.score);
     return (
       <TouchableWithoutFeedback onPress={this.handlePress}>
         <View style={{ width }}>
-          <HotelDetailPreview availability={data} />
+          <HotelDetailPreview
```

Comment:
What's the benefit of making explicit all the props instead of passing one object?